lib/html/pipeline/@mention_filter.rb

```diff
@@ -70,7 +70,7 @@ def self.mentioned_logins_in(text, username_pattern=UsernamePattern)
       def call
         result[:mentioned_usernames] ||= []
 
-        doc.search('text()').each do |node|
+        doc.search('.//text() | text()').each do |node|
```

Comment:
Why this change here? Since the tests pass, the behavior is the same, but I'm curious if this is the preferred convention.